### PR TITLE
Initial merge

### PR DESCRIPTION
**Pull request by johanneslosch**
The bot development state atm should be documented, after merging I will perform a command to host the docs via Github Pages

### CODE REVIEW

**Comment by Steffan153** [comment]
Wow that's a crazy amount of commits :joy:

**Comment by johanneslosch** [comment]
yeah but followable

File: docs/commands.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+### uptime
+Shows the time where the bot is running for.
+#### aliases
+ - up
+#### permissions
+ - user
+
+---
+### stop
+Stops the bot and trys to restart.
```
Comment (drillskibo) [inline docs/commands.md:12]: tries*

File: docs/commands.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Commands
+
+### uptime
+Shows the time where the bot is running for.
```
Comment (drillskibo) [inline docs/commands.md:4]: Shows since how long the bot has been running for.

File: docs/commands.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+#### aliases
+ - exit
+ - shutdown
+ - restart
+#### permissions
+ - owner
+
+ ---
+### ping
+Display the latency between server and bot.
```
Comment (drillskibo) [inline docs/commands.md:22]: Displays* the latency between the* server and the* bot.

File: docs/commands.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+#### aliases
+ - p
+ - pong
+#### permissions
+ - user
+
+ ---
+### help
+Send an information message with commands ith descriiption.
```
Comment (drillskibo) [inline docs/commands.md:31]: Sends an information message with the list of commands and their aliases.*

File: docs/commands.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+#### aliases
+ - h
+#### permissions
+ - user
+
+ ---
+### info
+Send an information message content about the bot and the guild.
```
Comment (drillskibo) [inline docs/commands.md:39]: Sends an information message about the bot and its data.

File: docs/commands.md (hunk 3 above)
Comment (johanneslosch) [reply docs/commands.md]: done in [047418f](https://github.com/PopperDevs/PopperBot-docs/pull/1/commits/047418f042be50d2f4bfec852262077c77afaea0)

File: docs/commands.md (hunk 4 above)
Comment (johanneslosch) [reply docs/commands.md]: done in [047418f](https://github.com/PopperDevs/PopperBot-docs/pull/1/commits/047418f042be50d2f4bfec852262077c77afaea0)

File: docs/commands.md (hunk 5 above)
Comment (johanneslosch) [reply docs/commands.md]: done in [047418f](https://github.com/PopperDevs/PopperBot-docs/pull/1/commits/047418f042be50d2f4bfec852262077c77afaea0)

File: docs/commands.md (hunk 2 above)
Comment (johanneslosch) [reply docs/commands.md]: done in [047418f](https://github.com/PopperDevs/PopperBot-docs/pull/1/commits/047418f042be50d2f4bfec852262077c77afaea0)

File: docs/commands.md (hunk 1 above)
Comment (johanneslosch) [reply docs/commands.md]: done in [047418f](https://github.com/PopperDevs/PopperBot-docs/pull/1/commits/047418f042be50d2f4bfec852262077c77afaea0)

**Comment by johanneslosch** [comment]
@drillskibo I fixed the typos in [047418f](https://github.com/PopperDevs/PopperBot-docs/pull/1/commits/047418f042be50d2f4bfec852262077c77afaea0)

**Pull request merged**